python_modules/dagster/dagster/core/storage/event_log/base.py

```diff
@@ -47,6 +49,51 @@ class EventLogRecord(NamedTuple):
     event_log_entry: EventLogEntry
 
 
+class AssetEntry(
+    NamedTuple(
+        "_AssetEntry",
+        [
+            ("asset_key", AssetKey),
+            ("last_materialization", Optional[Union[AssetMaterialization, EventLogEntry]]),
+            ("last_run_id", Optional[str]),
+            ("asset_details", Optional[AssetDetails]),
+            ("wipe_timestamp", Optional[datetime]),
+            ("last_materialization_timestamp", Optional[datetime]),
+            ("tags", Optional[Dict[str, str]]),
+        ],
+    )
+):
+    def __new__(
+        cls,
+        asset_key: AssetKey,
+        last_materialization: Optional[Union[AssetMaterialization, EventLogEntry]] = None,
+        last_run_id: Optional[str] = None,
+        asset_details: Optional[AssetDetails] = None,
+        wipe_timestamp: Optional[datetime] = None,
```

Comment:
Appreciate the input prha 🙌 

Removing the unnecessary columns now. Writing into the last_run_id column makes sense to me since we're already writing to it anyway, though it seems roughly the same to put it in AssetDetails. I don't feel strongly about it either way